deployment-bundle-steps.md

```diff
@@ -0,0 +1,39 @@
+# How to setup a project with deployment to S3
+
+This document assumes you have already setup the S3 bucket.
+
+## Using [CircleCi](https://circleci.com/)
+
+1. Log into [circleci.com](https://circleci.com/) and add your project. Set up the following credentials: (see other members of the team for these values).
+  - Environment Variables
+    - GHUSER
+    - GHPASS
```

Comment:
nice that you got rid of all the other controversial bits, I would still like someone who has more experience than me with this to confirm that this is the best thing to do, to put GHPASS as an env var in circle.